codersdk/provisionerdaemons.go

```diff
@@ -194,7 +194,12 @@ type ServeProvisionerDaemonRequest struct {
 // implementation. The context is during dial, not during the lifetime of the
 // client. Client should be closed after use.
 func (c *Client) ServeProvisionerDaemon(ctx context.Context, req ServeProvisionerDaemonRequest) (proto.DRPCProvisionerDaemonClient, error) {
-	serverURL, err := c.URL.Parse(fmt.Sprintf("/api/v2/organizations/%s/provisionerdaemons/serve", req.Organization))
+	orgParam := req.Organization.String()
+	if req.Organization == uuid.Nil {
+		orgParam = DefaultOrganization
```

Comment:
I don't get why we don't like `uuid.Nil` meaning the default on the API, but are fine with it on the SDK.

Reply:
I went back and forth on this. I think we are the only users of our sdk atm, but if I was to require the OrgID argument in the sdk to be set now, that would be a breaking change for the sdk.

I've been trying to maintain that single org deployments do not have to make any changes, and they continue to work as they do today. 

On the sdk side, this is an omission of a field being set.

On the API, the nil uuid could be an explicit value being set, rather than an omission.